packages/host/app/services/matrix-service.ts

```diff
@@ -53,6 +53,7 @@ import type {
   CardFragmentContent,
 } from 'https://cardstack.com/base/room';
 
+import { type ApplyButtonState } from '../components/ai-assistant/apply-button';
```

Comment:
so it looks like we are only holding this state in the current js runtime. if you reload the page i think we'll lose the apply state, right?